### PR TITLE
Dashboard return button.

### PR DESCRIPTION
**Pull request by jcs090218**
Implementation for #170.

### CODE REVIEW

File: dashboard.el [hunk 1]
```diff
@@ -125,6 +125,30 @@ Optional prefix ARG says how many lines to move; default is one line."
...
+        (setq fd-cnt (1+ fd-cnt))
+        (unless (= start-ln (line-number-at-pos))
+          (setq diff-line t)))
+      (unless (= (point) (point-max))
+        (setq entry-pt (point))))
+    (when (= fd-cnt 1)
+      (setq entry-pt (1- (point))))
+    (if entry-pt
+        (widget-button-press entry-pt)
+      (call-interactively #'widget-button-press))))
```
Comment (jcs090218) [inline dashboard.el]: Down compatible, so it still calls this function no matter what.

File: dashboard.el [hunk 2]
```diff
@@ -125,6 +125,30 @@ Optional prefix ARG says how many lines to move; default is one line."
...
+                  (not (= (point) (point-min)))
+                  (not (get-char-property (point) 'button))
+                  (not (= (point) (point-max))))
+        (forward-char 1)
+        (setq fd-cnt (1+ fd-cnt))
+        (unless (= start-ln (line-number-at-pos))
+          (setq diff-line t)))
+      (unless (= (point) (point-max))
+        (setq entry-pt (point))))
+    (when (= fd-cnt 1)
```
Comment (jcs090218) [inline dashboard.el]: This check if end of line.

File: dashboard.el [hunk 3]
```diff
@@ -125,6 +125,30 @@ Optional prefix ARG says how many lines to move; default is one line."
...
+  "Hit return key in dashboard buffer."
+  (interactive)
+  (let ((start-ln (line-number-at-pos))
+        (fd-cnt 0)
+        (diff-line nil)
+        (entry-pt nil))
+    (save-excursion
+      (while (and (not diff-line)
+                  (not (= (point) (point-min)))
+                  (not (get-char-property (point) 'button))
```
Comment (jcs090218) [inline dashboard.el]: Looking fore the widget button.

**Comment by JesusMtnez** [comment]
@jcs090218 Thank you so much! It looks pretty good! I'll test it later :+1:

**Pull request merged**